docs/guides/build-using-too-many-resources.rst

```diff
@@ -23,3 +23,20 @@ If there are extra packages that you don't need for building docs,
 you can create a custom requirements file just for documentation.
 This should speed up your documentation builds,
 as well as reduce your memory footprint.
+
+Use pip when possible
+---------------------
+
+In case you are using `conda` to create your environment,
+you could consider using `pip` instead since `conda` could require too much memory to calculate the dependency tree.
+
+Use system site-packages for pre-installed libs
+-----------------------------------------------
+
+There are a few libraries that Read the Docs has already installed (scipy, numpy, matplotlib, pandas, etc)
+in the Docker image used to build your docs. You can check the updated list of pre-installed libraries in the `Docker image repository`_.
+
+To use these pre-installed libraries and avoid consuming time re-downloading/compiling them,
+you ca use the :ref:`yaml-config:Python-use-system-site-packages` option to have access to them.
```

Comment:
The ref should be `yaml-config:python.use_system_site_packages` https://github.com/rtfd/readthedocs.org/pull/4422